models/public/db.js

```diff
@@ -11,15 +11,16 @@ const sequelize = new Sequelize(env.DATABASE_NAME, env.DATABASE_USERNAME, env.DA
     underscored: true,
     schema: env.SCHEMA
   },
-  dialectOptions: {
+  //Removed to test local database without ssl
+  /*dialectOptions: {
     ssl: {
-      require: true,
-      rejectUnauthorized: false
+      require: false,
+      rejectUnauthorized: true
```

Comment:
Don't remove ssl verification

Reply:
Removed for using local server without SSL verification.